apps/voting-disputable/contracts/DisputableVoting.sol

```diff
@@ -966,14 +961,22 @@ contract DisputableVoting is IForwarderWithContext, DisputableAragonApp {
     }
 
     /**
-    * @dev Tell if a vote's overrule window has started or not
+    * @dev Tell if a vote's delegated voting period has finished or not
     *      This function doesn't ensure whether the vote is open or not
     * @param _vote Vote instance being queried
     * @param _setting Setting instance applicable to the vote
-    * @return True if the vote's overrule window has started
+    * @return True if the vote's delegated voting period has finished
     */
-    function _hasStartedOverruleWindow(Vote storage _vote, Setting storage _setting) internal view returns (bool) {
-        return getTimestamp() >= _durationStartDate(_vote, _setting.overruleWindow);
+    function _hasFinishedDelegatedVotingPeriod(Vote storage _vote, Setting storage _setting) internal view returns (bool) {
+        uint64 pausedAt = _vote.pausedAt;
+        uint64 baseDelegatedVotingPeriodEndDate = _vote.startDate.add(_setting.delegatedVotingPeriod);
+
+        // If the vote was paused before the deleted voting period ends, we need to extend it
+        uint64 actualDeletedVotingEndDate = pausedAt != 0 && pausedAt < baseDelegatedVotingPeriodEndDate
+            ? baseDelegatedVotingPeriodEndDate.add(_vote.pauseDuration)
```

Comment:
You're going to get sick of these notes, but a minor optimization here is to load `pauseDuration` along with `pausedAt`.

Note that actually these are not in the same slot at the moment (but they should be—more on this in a comment later), but once they are, accessing `_vote.pausedAt` will give us `pauseDuration` almost for free.

Reply:
See 75ede7c.